.travis.yml

```diff
@@ -7,7 +7,7 @@ install:
     # see:
     #   https://github.com/nh2/static-haskell-nix/pull/27#issuecomment-502652181
     #   https://github.com/nixos/nix/issues/2733
-  - (for i in {1..5}; do bash <(curl https://nixos.org/nix/install) && exit 0; done; exit 1)
+  - curl -L https://nixos.org/nix/install | sh
```

Comment:
Need to remove the comment above as well. Issue 2733 is fixed in nix 2.3.3 and travis installs 2.3.6

Reply:
Fixed in a20807b8a3573c4c93ae90a0a2ccf083aad05f8c